packages/riipen-ui/src/components/Accordion.test.jsx

```diff
@@ -0,0 +1,150 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Accordion from "./Accordion";
+import AccordionDetails from "./AccordionDetails";
+import AccordionSummary from "./AccordionSummary";
+
+describe("<Accordion>", () => {
+  it("renders without errors", () => {
+    let error;
+    const child = <AccordionDetails />;
+
+    try {
+      mount(<Accordion>{child}</Accordion>);
+    } catch (e) {
+      error = e;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const summary = <AccordionSummary />;
+    const child = <AccordionDetails />;
+
+    const wrapper = mount(
+      <Accordion>
+        {summary}
+        {child}
+      </Accordion>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("default props", () => {
+    it("sets correct default props", () => {
+      const child = <AccordionDetails />;
+
+      const wrapper = mount(<Accordion>{child}</Accordion>);
+
+      expect(wrapper.find("Accordion").props().classes).toEqual([]);
+      expect(wrapper.find("Accordion").props().defaultExpanded).toEqual(false);
+      expect(wrapper.find("Accordion").props().disabled).toEqual(false);
+    });
+  });
+
+  describe("children prop", () => {
+    it("renders given children", () => {
+      const child = <AccordionDetails>hello</AccordionDetails>;
+
+      const wrapper = mount(<Accordion>{child}</Accordion>);
+
+      expect(wrapper.find("AccordionDetails").props().children).toEqual(
+        "hello"
+      );
+    });
+  });
```

Comment:
I don't think this test is right. We aren't testing something is being rendered. We are essentially doing:

1. Give AccordionDetails a child
2. Test did we give according details a child

I think a better test could be "does wrapped contain the child node".